dml.go

```diff
@@ -286,6 +286,8 @@ func insertSQL(columns []ColumnType, count int) string {
 				}
 				if unique.HasConflictEntry(indexRow) {
 					if tryTime >= RETRY_COUNT {
+						fmt.Println("retry time exceed RETRY_COUNT:", tryTime)
+						fmt.Println("index name:", unique.name, "data type:", unique.cols,"value:", indexRow, "duplicate")
```

Comment:
We can print this once a conflict entry found, not only when retry times exceed.